src/data/de/index.ts

```diff
@@ -328,10 +329,10 @@ export const loggedInData = {
       noNotifications: "abbestellen"
     },
     revisions: {
-      accept: {
-        action: "Akzeptieren",
-        title: "Bearbeitung akzeptieren",
-        explanation: "Bitte gib der Autor*in ein bisschen Feedback."
+      checkout: {
+        action: 'Accept',
+        title: 'Accept Revision',
+        explanation: 'Please give the author some feedback.'
```

Comment:
Reset to the english version?!

Reply:
yes, I changes the source string name.

Reply:
okay, strings are as they should be now.